src/StripeClient.Cards.cs

```diff
@@ -56,7 +56,7 @@ public StripeObject UpdateCard(string customerOrRecipientId, string cardId, ICre
 
             var request = new RestRequest();
             request.Method = Method.POST;
-            request.Resource = string.Format("{0}/{customerOrRecipientId}/cards/{cardId}", isRecipient ? "recipients" : "customers");
+            request.Resource = string.Format("{0}/{{customerOrRecipientId}}/cards/{cardId}", isRecipient ? "recipients" : "customers");
```

Comment:
@JasonS please update {cardId} with the same change "{{cardId}}", similarly for the rest of the updates for this PR